extensions/wikia/AdEngine/js/AdTracker.js

```diff
@@ -7,6 +7,11 @@ define('ext.wikia.adEngine.adTracker', ['wikia.tracker', 'wikia.window'], functi
 
 	function encodeAsQueryString(extraParams) {
 		var out = [], key, keys = [], i, len;
+
+		if (window.ads.context.opts.sp) {
```

Comment:
How different would it be to use `ext.wikia.adEngine.adContext` dependency instead of reaching for global `ads.context.opts.sp`?